src/lib/__tests__/makeParser.spec.ts

```diff
@@ -1,5 +1,9 @@
 import makeParser, { ParseError } from '../makeParser'
 
+global.console = {
+  error: jest.fn(),
+} as unknown as Console
```

Comment:
I'll double check, but jest creates a brand new env per test run. So this won't leak to other tests. I'll double check and report.

If this is the case, and jest isolates thus global to only this test, then you good with this?

Reply:
Oh, this is my mistake. For some reason, I thought this was a global mock.

As an alternative, I believe you can do something like this, to avoid the type assertions and removing other console methods:

```javascript
jest.spyOn(global, 'error').mockImplementation(() => {})
```